src/extension.ts

```diff
@@ -323,9 +324,7 @@ export function activate(context: vscode.ExtensionContext): void {
     });
     registerCommand('appService.AddCosmosDBConnection', addCosmosDBConnection);
     registerCommand('appService.RemoveCosmosDBConnection', removeCosmosDBConnection);
-    registerCommand('appService.revealConnection', () => {
-        throw new Error('Not implemented yet.');
-    });
+    registerCommand('appService.RevealConnection', async (node: CosmosDBDatabase) => vscode.commands.executeCommand('cosmosDB.revealTreeItem', node.connectionId));
```

Comment:
The cosmos command should have `.api.` in the name just like the other api calls